docs/source/hyper-parameter-search.rst

```diff
@@ -21,6 +23,10 @@ In that case, why use Dask-ML's versions?
 - :ref:`Works well with Dask collections <works-with-dask-collections>`. Dask
   arrays, dataframes, and delayed can be passed to ``fit``.
 
+- :ref:`State-of-the-art model selection algorithms <hyperband>`. We have
```

Comment:
Avoid the use of "we" here when discussing positive things.  It makes it sound like we're proud.

Reply:
Looks like my background is coming through... Academics use "we" all the time. I've removed it.